lib/plantain/traject/ead2_config.rb

```diff
@@ -62,9 +62,11 @@
 each_record do |_record, context|
   next unless settings["repository"]
 
-  context.clipboard[:repository] = Arclight::Repository.find_by(
```

Comment:
I'm not sure what problem this change solves. Would it be relevant to add a test? I'm not sure what a `context.cllipboard` is.

Reply:
There were cases where `Arclight::Repository.find_by` returned a `nil`, and this triggered errors for indexing where the `Repository` had not been created.